src/Components/CaseStatusBar/styles.ts

```diff
@@ -0,0 +1,45 @@
+import { StyleSheet } from 'react-native';
+
+import { colors } from '../../styles/colors';
+export default StyleSheet.create({
+  caseStatusContainer: {
+    marginTop: 20,
```

Comment:
There is a `rowGap` of 15px between every component on `CaseScreen` so let's take that into account when defining `marginTop`